### ISSUE TITLE
Error report -> UPLink Network -> Issue Reading Configuration

### ISSUE THREAD

**Issue by m4t7e0** [canonical/lxd-ui#757]
After Launch of microcloud setup, i had some problem with the UPLINK network, i can't read any information about this Network via CLI.

Error

Could not find physical in networkTypeToOptionKey

```
eG@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:87:30823
ie@https://172.16.14.185:8443/ui/assets/FormMenuItem-f4d64986.js:1:5989
te@https://172.16.14.185:8443/ui/assets/FormMenuItem-f4d64986.js:1:4865
be@https://172.16.14.185:8443/ui/assets/FormMenuItem-f4d64986.js:1:7867
J@https://172.16.14.185:8443/ui/assets/NetworkForm-34d3abf4.js:1:4879
Zg@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:38:19529
B1@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:40:44027
L1@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:40:39765
m$@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:40:39693
Cd@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:40:39545
Ym@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:40:35913
F0@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:40:36716
hi@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:38:3274
Dn/<@https://172.16.14.185:8443/ui/assets/index-b3f3eaed.js:40:34246

```
from cli 
~$ lxc network info UPLINK
Error: Network interface "UPLINK" not found
~$ lxc network list
+---------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
|     NAME      |   TYPE   | MANAGED |      IPV4       |           IPV6            | DESCRIPTION | USED BY |  STATE  |
+---------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
| test          | bridge   | YES     | 10.59.216.1/24  | fd42:88e8:3ae9:adaa::1/64 |             | 0       | CREATED |
+---------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
| UPLINK        | physical | YES     |                 |                           |             | 1       | CREATED |
+---------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
| bond0         | bond     | NO      |                 |                           |             | 0       |         |
+---------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
| bond0.174      | vlan     | NO      |                 |                           |             | 0       |         |
+---------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
| br-int        | bridge   | NO      |                 |                           |             | 0       |         |
+---------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
| default       | ovn      | YES     | 10.233.146.1/24 | fd42:c097:9e29:5557::1/64 |             | 6       | CREATED |
+---------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+
| lxdovn1       | bridge   | NO      |                 |                           |             | 0       |         |
+---------------+----------+---------+-----------------+---------------------------+-------------+---------+---------+

**Comment by edlerd** [comment]
Thank you for the report.

We can't edit or browse configuration for non-managed networks from lxd, hence we don't link to the network configuration page in the UI for physical networks. Did you manually change the URL to open the network configuration in the UI and that produced the error?

**Comment by m4t7e0** [comment]
Strange, UPLINK (physical) is managed by lxd, i just followed the microcloud guide, durind the setup i set the bond0 inteface, and the in lxd i saw this uplink network... when i tried to get more informatio via UI (just surfing the :8443) i got this error.

**Comment by edlerd** [comment]
You are right, it is managed in your case. I was under the wrong impression physical networks are never managed. Thanks for explaining your use case.

The linked PR above should fix the underlying issue.
